tests/unit/test_charm.py

```diff
@@ -248,6 +248,7 @@ def test_owasp_modsecurity_crs(self):
         # Test undefined.
         conf_or_rel = self.harness.charm._all_config_or_relations[0]
         self.assertEqual(conf_or_rel._owasp_modsecurity_crs, False)
+        self.assertEqual(conf_or_rel._owasp_modsecurity_custom_rules, "")
```

Comment:
Should the test description be updated to include information about that a new check has been included?

Reply:
Changed for "Test the owasp-modsecurity-crs and owasp-modsecurity-custom-rules properties."